src/main/java/seedu/planner/model/ModelManager.java

```diff
@@ -48,12 +49,12 @@ public ModelManager(ReadOnlyModulePlanner modulePlanner, UserPrefs userPrefs) {
 
         versionedModulePlanner = new VersionedModulePlanner(modulePlanner);
 
-        takenModules = new ArrayList<>();
+        takenModulesPerSemester = new ArrayList<>();
         for (int i = 0; i < MAX_NUMBER_SEMESTERS; i++) {
-            takenModules.add(new SortedList<>(versionedModulePlanner.getTakenModules(i), (x, y) ->
-                    x.compareTo(y)));
+            takenModulesPerSemester.add(new SortedList<>(versionedModulePlanner.getTakenModules(i)));
```

Comment:
Are the taken modules still sorted? Cos I removed the `x.compareTo(y)` and it wasn't sorted leh

Reply:
Oh ya just realised, sorry I'll put it back haha